sentence_transformers/evaluation/BinaryClassificationEvaluator.py

```diff
@@ -275,11 +276,15 @@ def compute_metrices(self, model: SentenceTransformer) -> dict[str, dict[str, fl
             f1, precision, recall, f1_threshold = self.find_best_f1_and_threshold(scores, labels, greater_is_better)
             ap = average_precision_score(labels, scores * (1 if greater_is_better else -1))
 
+            predicted_labels = (scores >= f1_threshold) if greater_is_better else (scores <= f1_threshold)
+            mcc = matthews_corrcoef(labels, predicted_labels)
+
             logger.info(f"Accuracy with {name}:           {acc * 100:.2f}\t(Threshold: {acc_threshold:.4f})")
             logger.info(f"F1 with {name}:                 {f1 * 100:.2f}\t(Threshold: {f1_threshold:.4f})")
             logger.info(f"Precision with {name}:          {precision * 100:.2f}")
             logger.info(f"Recall with {name}:             {recall * 100:.2f}")
             logger.info(f"Average Precision with {name}:  {ap * 100:.2f}\n")
+            logger.info(f"Matthews Correlation with {name}: {mcc:.4f}\n")
```

Comment:
Hi @JINO-ROHIT ,  
I think you could also add this output to the initial documentation of the BinaryClassificationEvaluator class.

Reply:
done!

Reply:
Good suggestion!